tutorials/07-searchlight/run_searchlight.sh

```diff
@@ -3,9 +3,9 @@
 
 #SBATCH --output=../logs/searchlight-%j.out
 #SBATCH --job-name searchlight
-#SBATCH -t 30        # time limit: how many minutes 
-#SBATCH -m=4G        # memory limit
-#SBATCH -n 2         # how many cores to use
+#SBATCH --t 30        # time limit: how many minutes 
+#SBATCH -mem=4G        # memory limit
+#SBATCH --n 2         # how many cores to use
```

Comment:
Multi-character options use two dahes, single-character options use a single dash. According to the manual, we should use `-t, -n, --mem`.